templates/docs/examples/patterns/chip/variants.html

```diff
@@ -366,364 +366,364 @@
     <span class="p-chip__value">Bob</span>
   </button>
 
-  <span class="p-chip--caution is-dark is-dense">
+  <span class="p-chip--caution is-dense">
     <span class="p-chip__lead">Owner</span>
     <span class="p-chip__value">Bob</span>
     <button class="p-chip__dismiss">
-      <i class="p-icon--close is-light">Dismiss</i>
+      Dismiss
     </button>
   </span>
 
   <br>
 
-  <button class="p-chip--negative is-dark is-dense">
+  <button class="p-chip--negative is-dense">
     <span class="p-chip__value">21.10</span>
   </button>
 
-  <span class="p-chip--negative is-dark is-dense">
+  <span class="p-chip--negative is-dense">
     <span class="p-chip__value">21.10</span>
     <button class="p-chip__dismiss">
-      <i class="p-icon--close is-light">Dismiss</i>
+      Dismiss
     </button>
   </span>
 
-  <button class="p-chip--negative is-dark is-dense">
+  <button class="p-chip--negative is-dense">
     <span class="p-chip__lead">Owner</span>
     <span class="p-chip__value">Bob</span>
   </button>
 
-  <span class="p-chip--negative is-dark is-dense">
+  <span class="p-chip--negative is-dense">
     <span class="p-chip__lead">Owner</span>
     <span class="p-chip__value">Bob</span>
     <button class="p-chip__dismiss">
-      <i class="p-icon--close is-light">Dismiss</i>
+      Dismiss
     </button>
   </span>
 
   <br>
 
-  <button class="p-chip--information is-dark is-dense">
+  <button class="p-chip--information is-dense">
     <span class="p-chip__value">21.10</span>
   </button>
 
-  <span class="p-chip--information is-dark is-dense">
+  <span class="p-chip--information is-dense">
     <span class="p-chip__value">21.10</span>
     <button class="p-chip__dismiss">
-      <i class="p-icon--close is-light">Dismiss</i>
+      Dismiss
     </button>
   </span>
 
-  <button class="p-chip--information is-dark is-dense">
+  <button class="p-chip--information is-dense">
     <span class="p-chip__lead">Owner</span>
     <span class="p-chip__value">Bob</span>
   </button>
 
-  <span class="p-chip--information is-dark is-dense">
+  <span class="p-chip--information is-dense">
     <span class="p-chip__lead">Owner</span>
     <span class="p-chip__value">Bob</span>
     <button class="p-chip__dismiss">
-      <i class="p-icon--close is-light">Dismiss</i>
+      Dismiss
     </button>
   </span>
 
   <br>
 
-  <button class="p-chip is-dark is-dense">
+  <button class="p-chip is-dense">
     <span class="p-chip__value">21.10</span>
   </button>
 
-  <button class="p-chip is-dark">
+  <button class="p-chip">
     <span class="p-chip__value">21.10</span>
   </button>
 
-  <span class="p-chip--positive is-dark is-dense">
+  <span class="p-chip--positive is-dense">
     <span class="p-chip__value">21.10</span>
     <button class="p-chip__dismiss">
-      <i class="p-icon--close is-light">Dismiss</i>
+      Dismiss
     </button>
   </span>
 
-  <span class="p-chip--positive is-dark">
+  <span class="p-chip--positive ">
```

Comment:
Can we remove the trailing whitespace for consistency?

Reply:
Yes, good catch. Must be some leftover from search/replace.